test/integration/sagemaker/test_spark.py

```diff
@@ -210,6 +210,66 @@ def test_sagemaker_pyspark_sse_s3(role, image_uri, sagemaker_session, region, sa
     assert len(output_contents) != 0
 
 
+def test_sagemaker_pyspark_sse_kms_s3(role, image_uri, sagemaker_session, region, sagemaker_client, configuration):
+    spark = PySparkProcessor(
+        base_job_name="sm-spark-py",
+        image_uri=image_uri,
+        role=role,
+        instance_count=2,
+        instance_type="ml.c5.xlarge",
+        max_runtime_in_seconds=1200,
+        sagemaker_session=sagemaker_session,
+    )
+
+    kms_key_id = None
+    kms_client = sagemaker_session.boto_session.client("kms", region_name=region)
+    for alias in kms_client.list_aliases()["Aliases"]:
+        if "s3" in alias["AliasName"]:
+            kms_key_id = alias["TargetKeyId"]
+
+    if not kms_key_id:
+        raise ValueError("AWS managed s3 kms does not exist")
+
+    bucket = sagemaker_session.default_bucket()
+    timestamp = datetime.now().isoformat()
+    input_data_key = f"spark/input/sales/{timestamp}/data.jsonl"
+    input_data_uri = f"s3://{bucket}/{input_data_key}"
+    output_data_uri_prefix = f"spark/output/sales/{timestamp}"
+    output_data_uri = f"s3://{bucket}/{output_data_uri_prefix}"
+    s3_client = sagemaker_session.boto_session.client("s3", region_name=region)
+    with open("test/resources/data/files/data.jsonl") as data:
+        body = data.read()
+        s3_client.put_object(
+            Body=body, Bucket=bucket, Key=input_data_key, ServerSideEncryption="aws:kms", SSEKMSKeyId=kms_key_id
+        )
+
+    spark.run(
+        submit_app="test/resources/code/python/hello_py_spark/hello_py_spark_app.py",
+        submit_py_files=["test/resources/code/python/hello_py_spark/hello_py_spark_udfs.py"],
+        arguments=["--input", input_data_uri, "--output", output_data_uri],
+        # configuration=configuration
+        configuration={
+            "Classification": "core-site",
+            "Properties": {
+                "fs.s3a.server-side-encryption-algorithm": "SSE-KMS",
+                "fs.s3a.server-side-encryption.key": f"arn:aws:kms:us-west-2:790336243319:key/{kms_key_id}",
```

Comment:
This ARN has the region `us-west-2` -- could you use the `region` pytest fixture in place of this?

Same for the account ID, which we have a fixture for (named `account_id`):

https://github.com/aws/sagemaker-spark-container/blob/e3dc9678a30b95aa367ab9e4cc112e25d131fdf8/test/integration/conftest.py#L37-L39

Could you pass that `account_id` into this test function and substitute the account ID here with the value of `account_id`?

Reply:
yep. will do.